get_instagram.py

```diff
@@ -0,0 +1,27 @@
+from InstagramAPI import InstagramAPI
+from key import user_name, password
+
+
+class GetInstagramInfo:
+    def __init__(self, username, password):
+	self.username = username
+	self.password = password
+	self.api = InstagramAPI(username=user_name, password=password)
+	self.api.login()
+	self.user_id = self.api.username_id
+
+    def get_followers(self):
+	"""Demonstrate followers of the profile"""
```

Comment:
Please change your docstring to make it more clear.

It might be something like this:

Returns list of user's followers.

Reply:
done